portal/app/settings/base.py

```diff
@@ -111,7 +112,9 @@
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'rest_framework.authentication.TokenAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
+        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
```

Comment:
I don' think this is needed anymore. You should be able to remove the JSON, Basic and the rest_framework_jwt.

Reply:
but we still need to figure out the authentication

Reply:
Sure, but it's not being figured out by this PR, or is it?

Reply:
well, maybe not... you are right.